ufo2fontir/src/source.rs

```diff
@@ -1252,11 +1258,12 @@ mod tests {
     fn add_design_location(
         add_to: &mut HashMap<PathBuf, Vec<DesignLocation>>,
         glif_file: &str,
-        axis: &str,
+        tag: &str,
```

Comment:
would it make sense to ask the caller to pass in a `Tag`?

Reply:
Yes, will fix.